### ISSUE TITLE
Question about batch norm

### ISSUE THREAD

**Issue by crawles** [matthias-wright/flaxmodels#15]
Thank you so much for creating this. For resnet, I am wondering how the implementation of BatchNorm differs from the Flax implementation? Basically, I'm wondering if I can replace `ops.BatchNorm` with `flax.linen.BatchNorm` to reduce dependencies? Thanks!

**Comment by matthias-wright** [comment]
Hi @crawles! The implementation is the same in terms of functionality. The only difference is that I added the attributes `var_init` and `mean_init` to `ops.BatchNorm`. This allows me to initialize a batch norm module with the mean and var stats from a checkpoint, as done in [this](https://github.com/matthias-wright/flaxmodels/blob/main/flaxmodels/resnet/ops.py#L30) line. This is important for when you initialize the model with `pretrained='imagenet'` as in [this](https://github.com/matthias-wright/flaxmodels/tree/main/flaxmodels/resnet#2-basic-usage) example. 
To answer your question, if you are interested in using the pretrained weights, you have to use `ops.BatchNorm`. When you just want to train your network from scratch, you can use `flax.linen.BatchNorm`.

**Comment by crawles** [comment]
Got it. Came here for the pretrained model so will be using `ops`. Thank you!
